problemsets-mat223/modules/module1.tex

```diff
@@ -479,7 +479,20 @@
 	Let $\vec x=\xhat-\yhat$, $\vec y=3\xhat-\yhat$, and $\vec r=2\xhat+2\yhat$. Is
 	$\vec r$ a linear combination of $\vec x$ and $\vec y$?
 
-	XXX Finish
+	Assume $\vec r$ is a linear combination of $\vec x$ and $\vec y$ for some unknown coefficients $\alpha_1$ and $\alpha_2$. Then
+	\begin{align*}
+	    \vec r&=\alpha_1\vec x+\alpha_2\vec y\\
+	   \implies 2\xhat+2\yhat&=\alpha_1(\xhat-\yhat)+\alpha_2(3\xhat-\yhat)\\
+	   \implies 2\xhat+2\yhat&=(\alpha_1+3\alpha_2)\xhat+(-\alpha_1-\alpha_2)\yhat.
+	\end{align*}
+	Thus, it is equivalent to solve the two-equations system
+	\begin{equation}
+	    \sysdelim\{.
+		\systeme{
+			\alpha_1+3\alpha_2=2,    % I have no idea why \alpha_1 would not compile out in my computer, I
```

Comment:
Use $\alpha$ and $\beta$. I think the subscripts confuse `\systeme`.

Reply:
Looks like \beta is also not able to use. I guess in system package we are not allowed to use any "\xxx" for the first variable. I will use a and b instead. or a_1 and a_2